code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -141,6 +136,13 @@
 	update_icon()
 	return
 
+/mob/living/simple_animal/bot/mulebot/screwdriver_act(mob/living/user, obj/item/I)
+	. = ..()
+	if(open)
+		on = FALSE
+	update_controls()
+	update_icon()
+
```

Comment:
will this return TRUE? I'm not sure what the behaviour of `. = ..()` will return here

Reply:
Yeah, that should have been converted to return TRUE. . = ..() was an accident